### DISCUSSION TITLE
custom presets on android tv

### DISCUSSION THREAD

**Discussion by mateuszaaa** [projectM-visualizer/projectm#549] [Q&A] [answered]
Hello, is it possible to use projectM on android tv with some custom presets ?

**Comment by OfficialIncubo** [comment]
I am trying to install projectM TV apk file on every website and didn't work. I don't want to buy projectM TV.

I have a single solution: You can easily share this app to TV using ChromeCast or something.
![Screenshot_20220124_152321.jpg](https://user-images.githubusercontent.com/74858188/150790649-a035c56b-a717-44dc-8044-64b3838f8121.jpg)

**Comment by kblaschke** [comment, marked as answer]
Sadly, the projectM dev team cannot provide support for the Android app, as it's not being written or maintained by us, but someone else (Peter Sperl). Please contact the author directly to get help on your issue.

**Comment by OfficialIncubo** [comment]
I have tried with my Android Projector called "Loosafe YG433".

It's possible. The path is the same, look at `storage/emulated/0/Android/data/com.psperl.projectMTV/files/presets_dir`
Don't put additional presets in the subfolder from the actual preset directory, otherwise it will crash quickly. Just put these presets and textures only on preset directory. I just looked the path months ago I don't know.

**Reply by mateuszaaa** [reply]
wow didnt expected solution to come! will try it out !
